pom.xml

```diff
@@ -48,18 +48,19 @@
         <dependency.freemarker.version>2.3.28</dependency.freemarker.version>
         <dependency.rhino.version>1.7.11</dependency.rhino.version>
         <dependency.httpcomponents.version>4.5.9</dependency.httpcomponents.version>
-        <dependency.alfresco-repository.version>7.185</dependency.alfresco-repository.version>
+        <dependency.alfresco-repository.version>7.193</dependency.alfresco-repository.version>
+        <dependency.alfresco-remote-api.version>7.193</dependency.alfresco-remote-api.version>
         <dependency.alfresco-jlan.version>7.1</dependency.alfresco-jlan.version>
         <dependency.alfresco-core.version>7.33</dependency.alfresco-core.version>
-        <dependency.alfresco-remote-api.version>7.134</dependency.alfresco-remote-api.version>
+        <dependency.alfresco-remote-api.version>7.144</dependency.alfresco-remote-api.version>
         <dependency.webscripts.version>7.15</dependency.webscripts.version>
         <dependency.surf.version>7.7</dependency.surf.version>
         <dependency.alfresco-sdk.version>2.0.0</dependency.alfresco-sdk.version>
         <dependency.dom4j.version>2.1.1</dependency.dom4j.version>
         <dependency.slf4j.version>1.7.30</dependency.slf4j.version>
         <dependency.cxf.version>3.3.5</dependency.cxf.version>
         <dependency.beanutils.version>1.9.4</dependency.beanutils.version>
-        <dependency.keycloak.version>8.0.1</dependency.keycloak.version>
```

Comment:
I see that keycloak is still at 9.0.3 in https://github.com/Alfresco/alfresco-repository/blob/support/SP/7.N/pom.xml. Did you check with Repo colleagues that 11.0.0-alfresco-001 is indeed the version to use?

Reply:
Yes the version is not in SP/7.N .  @jamalkm  will fix version for his task https://issues.alfresco.com/jira/browse/ACS-300?focusedCommentId=798740&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-798740. 
It will update, the final version will be : 11.0.0-alfresco-001.